src/components/views/elements/RoomAliasField.tsx

```diff
@@ -135,12 +135,13 @@ export default class RoomAliasField extends React.PureComponent<IProps, IState>
                         return true;
                     }
                     const split = value.split(':');
-                    if (split.length > 2) {
+                    // Define the value invalid if there's no first part (roomname) or separator's missing
+                    if (split.length < 2 || split[0].length < 1) {
```

Comment:
This duplicates a check from `hasDomain`, they both check if it has at least 1 `:` which seems duplicative and causes this

![image](https://user-images.githubusercontent.com/2403652/144406861-94f2f7d9-f7fb-4280-b3b4-9b41a43c22fa.png)